### PR TITLE
Made LS image time continuous

### PR DESCRIPTION
**Pull request by jessemapel**
We were having issues with ground partials being inconsistent across the image, this makes them consistent.

### CODE REVIEW

File: tests/LineScanCameraTests.cpp [hunk 1]
```diff
@@ -238,7 +238,7 @@ TEST_F(OrbitalLineScanSensorModel, InversionHeight) {
 TEST_F(OrbitalLineScanSensorModel, InversionReallyHigh) {
   for (double line = 0.5; line < 16; line++) {
     csm::ImageCoord imagePt(line, 8);
-    csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 49000.0);
```
Comment (jessemapel) [inline tests/LineScanCameraTests.cpp]: This was too close to the sensor, it caused the ground to image to diverge because of how close it was.

File: tests/LineScanCameraTests.cpp [hunk 2]
```diff
@@ -105,7 +105,7 @@ TEST_F(ConstVelocityLineScanSensorModel, calculateAttitudeCorrection) {
   double attCorr[9];
   adj.resize(15, 0);
   // Pi/2 with simply compensating for member variable m_flyingHeight in UsgsAstroLsSensorModel
-  adj[7] = (M_PI / 2) * 990.0496255790623081338708;
+  adj[7] = (M_PI / 2) * sensorModel->m_flyingHeight;
```
Comment (acpaquette) [inline tests/LineScanCameraTests.cpp]: Why this change? I guess, why was it hard coded in the first place?

Comment (jessemapel) [reply tests/LineScanCameraTests.cpp]: This was just hard-coded when the test was written, I think we forgot the sensor model state was public. I just happened to break it, so I made it more flexible.

**Comment by jessemapel** [comment]
This is obsoleted by #260 So, I'm closing it. If we ever have problems with that we can re-investigate this.
